bin/s2nc.c

```diff
@@ -80,10 +80,8 @@ void usage()
     fprintf(stderr, "    Set dynamic record timeout threshold\n");
     fprintf(stderr, "  -C,--corked-io\n");
     fprintf(stderr, "    Turn on corked io\n");
-    if (S2N_IN_TEST) {
```

Comment:
Do we actually want to keep this check? We probably shouldn't advertise TLS1.3 support.

Reply:
my preference would be to remove this check, it's not clear when using --tls13 on the command line to fails to enable tls13 because the env variable is not set (has bitten me and other testers in our experience).

Reply:
I guess that's fine then. The option does clearly label the feature "experimental".